libs/vkd3d-utils/vkd3d_utils_main.c

```diff
@@ -21,14 +21,14 @@
 #include "vkd3d_common.h"
 #include "vkd3d_utils_private.h"
 
-HRESULT WINAPI D3D12GetDebugInterface(REFIID iid, void **debug)
+VKD3D_UTILS_EXPORT HRESULT WINAPI D3D12GetDebugInterface(REFIID iid, void **debug)
```

Comment:
Does this actually make these symbols visible? The Meson build just defines VKD3D_UTILS_EXPORT to nothing.

Reply:
Difference between ``VKD3D_UTILS_EXPORTS`` and ``VKD3D_UTILS_EXPORT``.

This is a common Windows DLL export pattern.